tests/PhpcrUtils/CndWriterTest.php

```diff
@@ -86,18 +87,14 @@ function ($prefix) {
                         case 'ex':
                             return 'http://namespace.com/example';
                         default:
-                            throw new \Exception($prefix);
+                            throw new Exception($prefix);
                     }
                 }
             ))
         ;
         $cndWriter = new CndWriter($ns);
-        $res = $cndWriter->writeString(array($tpl));
+        $res = $cndWriter->writeString([$tpl]);
 
         $this->assertEquals($cnd, $res);
     }
 }
-
-abstract class MockNamespaceRegistry implements \Iterator, NamespaceRegistryInterface
```

Comment:
Doesn't it conflict with PSR-1: "This means each class is in a file by itself, and is in a namespace of at least one level: a top-level vendor name." ?

And in .styleci.yml we declared PSR-2 support and it says: "Code MUST follow a “coding style guide” PSR [PSR-1]."